core/azure-core/src/main/java/com/azure/core/polling/PollResponse.java

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.polling;
+
+import java.time.Duration;
+import java.util.Map;
+
+/**
+ * The life cycle of an operation depicted below
+ * <ul>
+ *    <li>NOT-STARTED </li>
+ *    <li>IN-PROGRESS</li>
+ *    <li>Successfully Complete</li>
+ *    <ul>User Cancelled</ul>
+ *    <ul>Failed</ul>
+ * </ul>
+ */
```

Comment:
Don't document the life cycle of an operation on the PollResponse class. It would more appropriately be covered on the OperationStatus enum.